### DISCUSSION TITLE
[Question]: how to delete docker-hosted frigate logs

### DISCUSSION THREAD

**Discussion by reassuringgreet** [blakeblackshear/frigate#10432] [Ask A Question] [answered]
### What is your question:

I have a huge log file due to some errors that happened, and every time I do "docker logs -f frigate" it's 5 minutes of scrolling. I know I can tail the logs, but I'd rather clean them and start fresh. Looking into how to delete the logs, I see that docker should return the logfile it's using with:

`docker container inspect  --format='{{.LogPath}}' frigate`

but it's not giving me any output. I tried other things like:

`sudo sh -c 'truncate -s 0 /var/lib/docker/containers/*/*-json.log'`

but nothing seems to work. Is there anyway to reset docker logs for frigate? Thanks

**Comment by NickM-27** [comment]
are you referring to the logs inside the frigate web UI or from docker itself?

**Comment by reassuringgreet** [comment]
sorry, should've clarified, to the logs I see with "docker logs -f frigate" (I believe they match the ones I see on the web UI in the frigate section, not Go2rtc nor Nginx)

**Reply by NickM-27** [reply]
docker logs -f frigate will show a combination of frigate, go2rtc, and nginx logs. I'd generally recommend just using the webUI to view logs.

**Reply by reassuringgreet** [reply]
right, but those logs are taking space somewhere on the server, do you know if there's any way to delete those? otherwise I believe I should backup frigate's config and recreate the container? not sure how to solve it

**Reply by NickM-27** [reply, marked as answer]
Frigate logs only take up space in memory unless the docker config was not done according to the docs. You'll have to look at your docker config on the host to figure out how large the logs are allowed to go

**Reply by reassuringgreet** [reply]
ohhh got it, thought they were going to disk. In that case I have no problem with the remaining logs. Thanks for the support.
